index.js

```diff
@@ -143,23 +178,47 @@ function init (options) {
   var base = nodePath.resolve(
     options.base || nodePath.join(__dirname, '../..')
   )
-  var packagePath = base.replace(/\/package\.json$/, '') + '/package.json'
 
-  try {
-    var npmPackage = require(packagePath)
-  } catch (e) {
-    // Do nothing
+  var packagePath = ''
+  var moduleAlias = null
```

Comment:
Rename it to `config`? I think that would be more clear what it is

Reply:
Renamed `packagePath` to `configPath`. Are you saying to rename `moduleAlias` to `config`?

If yes, yup that makes sense.